pycyphal/transport/commons/high_overhead_transport/_transfer_reassembler.py

```diff
@@ -708,22 +743,24 @@ def push(timestamp: Timestamp, frame: Frame) -> typing.Optional[TransferFrom]:
         is None
     )
     assert error_counters == {
+        ta.Error.INTEGRITY_ERROR: 1,
         ta.Error.MULTIFRAME_MISSING_FRAMES: 2,
         ta.Error.MULTIFRAME_EMPTY_FRAME: 1,
-        ta.Error.MULTIFRAME_INTEGRITY_ERROR: 1,
         ta.Error.MULTIFRAME_EOT_MISPLACED: 1,
         ta.Error.MULTIFRAME_EOT_INCONSISTENT: 1,
     }
 
     # Valid single-frame transfer with no payload.
     assert push(
         mk_ts(6000.0),
-        mk_frame(transfer_id=0, index=0, end_of_transfer=True, payload=b""),
-    ) == mk_transfer(timestamp=mk_ts(6000.0), transfer_id=0, fragmented_payload=[b""])
+        mk_frame(transfer_id=0, index=0, end_of_transfer=True, payload=b"" + TransferCRC.new(b"").value_as_bytes),
+    ) == mk_transfer(
+        timestamp=mk_ts(6000.0), transfer_id=0, fragmented_payload=[]
+    )  # fragmented_payload = [b""]?
```

Comment:
Semantically, `[]` is equivalent to `[b""]`.

Reply:
But not practically :\

![image](https://user-images.githubusercontent.com/39975120/223984945-69ee9885-afe6-4a97-b92d-916a617dd88c.png)